### PR TITLE
Added list of identifiers to UnifiedItem

### PR DESCRIPTION
**Pull request by alicefuzier**
Created first integration test for CalmService with embedded ElasticSearch. Removed source from UnifiedItem and added a list of Identifiers to it. Modified Record accordingly

### CODE REVIEW

File: api/src/test/scala/uk/ac/wellcome/platform/api/services/CalmServiceTest.scala [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import uk.ac.wellcome.finatra.services.ElasticsearchService
+import uk.ac.wellcome.models.{Identifier, UnifiedItem}
+import uk.ac.wellcome.platform.api.models.Record
+
+class CalmServiceTest extends AsyncFunSpec with ElasticSugar with Matchers{
+
+  val elasticSearchService = new ElasticsearchService(client)
+  val calmService = new CalmService(elasticSearchService)
+
+  it("should find records") {
```
Comment (kenoir) [inline api/src/test/scala/uk/ac/wellcome/platform/api/services/CalmServiceTest.scala]: ✨ ✨ tests!✨ ✨

File: transformer/src/main/scala/uk/ac/wellcome/transformer/actors/PublishableMessageRecordActor.scala [hunk 2]
```diff
@@ -34,7 +34,7 @@ class PublishableMessageRecordActor @Inject()(
       JsonUtil.toJson(unifiedItem) match {
         case Success(stringifiedJson) => {
           val message = SNSMessage(
-            Some(unifiedItem.source),
+            Some("Foo"),
```
Comment (kenoir) [inline transformer/src/main/scala/uk/ac/wellcome/transformer/actors/PublishableMessageRecordActor.scala]: Maybe `Some("calm")` to reduce levels of "Foo".

**Comment by kenoir** [comment]
@alicefuzier Merge and deploy is up to you :)

Pinging @jtweed to notify him that this will break the API (if we deploy that service) as the current data in the index does not match this model. 

This model is however more correct and the pipeline is _almost_ ready to run data through in the correct format!

**Comment by jtweed** [comment]
Fantastic thanks!

**Pull request merged**